src/main/java/lamball/TaskList.java

```diff
@@ -0,0 +1,198 @@
+package lamball;
+
+import lamball.task.Deadline;
+import lamball.task.Event;
+import lamball.task.Task;
+import lamball.task.ToDo;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+
+/**
+ * A class that handles the to-do list operations.
+ *
+ * @author ongzhili
+ */
+public class TaskList {
+    private final String INDENT = "    ____________________________________________________________\n";
+    protected ArrayList<Task> tasks;
+
+    String lastDoneTask;
+
+
+    /**
+     * Constructor for TaskList
+     *
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+        this.lastDoneTask = "";
+    }
+
+    private void printList(ArrayList<Task> lst) {
+        String listStr = "    Here aaaaare the taaaasks in your list:";
```

Comment:
Method named print, but no printing here? Is `getList()`, or `updateList()` a better name?